pkg/controller/job/job_controller.go

```diff
@@ -89,7 +90,8 @@ type JobController struct {
 	recorder record.EventRecorder
 }
 
-func NewJobController(podInformer coreinformers.PodInformer, jobInformer batchinformers.JobInformer, kubeClient clientset.Interface) *JobController {
+// NewController creates a new Controller.
```

Comment:
Again something more along the lines of "NewController returns a new job controller to keep the relevant pods in sync with the corresponding job objects"